packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -164,7 +164,7 @@ const renderHeaderContent = ({
             {`${stateLabel.toUpperCase()}`}
           </LabelWrapper>
           <VisuallyHiddenText {...hiddenTextProps}>
-            {isLive && ` ${stateLabel}`}
+            {stateLabel === 'Live' && ` ${stateLabel}`}
```

Comment:
Looks good to me. Just wondering if there is any difference between using `isLive` and `stateLabel === 'Live' `?

Reply:
`isLive` is used because a comma will be rendered for live programs for every service. And `stateLabel === 'Live'` is used so that the `stateLabel` is rendered only for some services that need "Live" pronounced correctly by a screen reader. For example, in RTL services for a live card `stateLabel === 'Live'` will be `false` because the label will be translated but `isLive` will be true